website/src/css/custom.css

```diff
@@ -399,32 +638,50 @@ nav.navbar div.navbar__inner {
   line-height: 26px;
 }
 
-/* Table of Contents */
-.table-of-contents__link:hover,
-.table-of-contents__link:hover code,
-.table-of-contents__link--active,
-.table-of-contents__link--active code {
-  color: var(--ifm-link-hover-color);
-}
-.table-of-contents {
-  font-size: 0.9rem;
+/* Pagination */
+.pagination-nav {
+  gap: 1em;
 }
 
-/* Pagination */
 .pagination-nav__link {
+  padding: var(--ifm-global-spacing) 0;
   transition: 100ms all ease-in-out;
+  border: none;
+}
+
+.pagination-nav__item--next .pagination-nav__label::after {
+  content:  var(--pagination-icon-next);
+  font-weight: var(--ifm-font-weight-narrow);
+  margin-left: .5em;
+}
+
+.pagination-nav__item:first-child .pagination-nav__label::before {
+  content:  var(--pagination-icon-prev) ;
+  font-weight: var(--ifm-font-weight-narrow);
+  margin-right: .5em;
+}
+
+.pagination-nav__sublabel {
+  font-weight: var(--ifm-font-weight-narrow);
+  margin-bottom: 1em;
 }
 
 /* Font Sizing Adjustments */
-.markdown > p {
-  font-size: 1.05rem;
+.markdown > p,
+.markdown li,
+blockquote {
+  font-size: clamp(.875rem, 1.5vw, 1.25rem);
+  line-height: 1.4;
 }
```

Comment:
Same as the footer text, I feel like the paragraph text within the docs is slightly too big. It was originally 16px on the live site, and looks like it's at 20px in this preview. Maybe meeting in the middle at `18px` would be the sweet spot?

Reply:
changed to 18px:
https://deploy-preview-1990--docs-getdbt-com.netlify.app/docs/introduction